doc/installation.rst

```diff
@@ -2,15 +2,25 @@ Installation
 ============
 
 
-Install ``can`` with ``pip``:
+Install the ``can`` package with ``pip``/``poetry`` or similar:
```

Comment:
I would remove poetry here as it is mostly used for development installations (afaik). This might be confusing for beginners. Devs hopefully know how to install packages.

Reply:
That's fair